server/routes/induction/exemption/exemption/index.ts

```diff
@@ -20,30 +22,36 @@ export default (router: Router, services: Services) => {
   const exemptionRecordedController = new ExemptionRecordedController()
 
   router.get('/prisoners/:prisonNumber/induction/exemption', [
+    checkUserHasPermissionTo(ApplicationAction.EXEMPT_INDUCTION),
```

Comment:
Call the new middleware `checkUserHasPermissionTo`
I quite like this - it is clear and expressive, saying "have you got permission to do this thing", rather than "do you have a specific role" 👍